apps/showcase/src/components/showcase/component-replacement/component-replacement-pres.template.html

```diff
@@ -0,0 +1,9 @@
+<div class="card p-2">
+  <div class="w-50">
+    <p>Gregorian date: <b>{{(dateFormControl.valueChanges | async) || initialDate}}</b></p>
```

Comment:
```suggestion
    <p>Gregorian date: <b>{{dateFormControl.value}}</b></p>
```
Isn't it working like this, as you initialize `dateFormControl` with `initialData` ?

Reply:
It's working but it;s not fancy :D 
updated :)